content/news/draft-bevy-0.14/index.md

```diff
@@ -5,7 +5,21 @@ date = 2024-05-17
 public_draft = 1188
 +++
 
-<!-- TODO Intro -->
+Thanks to **TODO** contributors, **TODO** pull requests, community reviewers, and our [**generous sponsors**](/community/donate), we're happy to announce the **Bevy 0.14** release on [crates.io](https://crates.io/crates/bevy)!
```

Comment:
I assume these TODO are intentionally left in?

Reply:
Yep, we can't get the final numbers until commits are frozen.